### PR TITLE
NMH-34: Add fields to filtering functionality [skip test]

### PR DESCRIPTION
**Pull request by KshitizGIT**
Jira Ticket [NMH-34](https://johnsnowlabs.atlassian.net/browse/NMH-34)

### CODE REVIEW

File: docs/_plugins/search_index.rb [hunk 1]
```diff
@@ -74,6 +74,11 @@ def download_link
     m ? m[1] : nil
   end
 
+  def model_type
```
Comment (pabla) [inline docs/_plugins/search_index.rb:77]: We need only two possible results here: `model` or `pipeline`.

**Review by pabla — CHANGES_REQUESTED** [review]

File: docs/_plugins/search_index.rb (hunk 1 above)
Comment (KshitizGIT) [reply docs/_plugins/search_index.rb:77]: what if the docs, don't have the Type ?

Comment (KshitizGIT) [reply docs/_plugins/search_index.rb]: Updated

**Review by pabla — APPROVED** [review]

**Comment by pabla** [comment]
All changes from this PR were applied in #6610

**Pull request closed without merging**